pkg/commands/objectstorage/accesskeys/get.go

```diff
@@ -0,0 +1,69 @@
+package accesskeys
+
+import (
+	"errors"
+	"io"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+	"github.com/fastly/go-fastly/v9/fastly/objectstorage/accesskeys"
+
+	"github.com/fastly/cli/pkg/argparser"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/global"
+	"github.com/fastly/cli/pkg/text"
+)
+
+// GetCommand calls the Fastly API to get an access key.
+type GetCommand struct {
+	argparser.Base
+	argparser.JSONOutput
+
+	// Required.
+	id string
```

Comment:
I would recommend to replace `id` with `key` or `access-key` everywhere in the implementation since `id` is not used anywhere in our public docs as well.